ui/src/containers/DisableAppModalContainer.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import { Button, TextInput, Form, FormGroup, Stack, StackItem, Modal } from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { toggleDisableAppModal } from '../actions/actions-ui';
+
+const DisableAppModalContainer = ({ isOpen, toggleDisableAppModal }) => {
+  function handleDisableAppSave () {
+    // TODO: This needs to send a request to the backend and update the client state
+    // this.props.toggleDisableAppModal();
+  };
+
+  const [disableMessage, setDisableMessage] = useState('');
+
+  return (
+    <Modal
+      isLarge
+      title="Disable All App Versions"
+      isOpen={isOpen}
```

Comment:
```suggestion
      isOpen={this.props.isOpen}
```
(not sure you can access it without `this.props`)

Reply:
On line 7 it is defined. From what I have found, this is the standard way of listing props in a functional component